plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -96,18 +97,36 @@ def updateJobs(FilePath filePath, String namePrefix = '') {
             if (firstLine.contains('View')) {
                 totalViews++
                 def view = parent.getView(name)
-                if (view) {
-                    updateView(view, name, file)
-                } else {
-                    createView(parent, name, file)
+                try {
+                    if (view) {
+                        updateView(view, name, file)
+                    } else {
+                        createView(parent, name, file)
+                    }
+                } catch (Exception ignored) {
+                    if (view) {
+                        println "Could not update ${fullName}"
```

Comment:
It would be nice to print at least the Exception message:

`println "Could not update ${fullName}: ${e.message}"

Same for the messages below.

Reply:
Yeah, you're right. Will update this.